docusaurus.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   onBrokenLinks: 'warn',
```

Comment:
above, url should be `bee-js.ethswarm.org`, note that we need to amend this (and base_url) automatically when we build for Bee in github actions